src/main/java/hlf/java/rest/client/service/impl/AddAnchorPeerToChannelWriteSetBuilderImpl.java

```diff
@@ -0,0 +1,91 @@
+package hlf.java.rest.client.service.impl;
+
+import hlf.java.rest.client.exception.ServiceException;
+import hlf.java.rest.client.model.AnchorPeerDTO;
+import hlf.java.rest.client.model.AnchorPeerParamsDTO;
+import hlf.java.rest.client.service.AddAnchorPeerToChannelWriteSetBuilder;
+import hlf.java.rest.client.util.FabricChannelUtil;
+import hlf.java.rest.client.util.FabricClientConstants;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.hyperledger.fabric.protos.common.Configtx.ConfigGroup;
+import org.hyperledger.fabric.protos.common.Configtx.ConfigValue;
+import org.hyperledger.fabric.protos.peer.Configuration.AnchorPeer;
+import org.hyperledger.fabric.protos.peer.Configuration.AnchorPeers;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
[nitpick] Would builder be more close to what is accomplished in this class?

Reply:
Sure, As discussed I have removed the private variables from the service and using it as argument to the method.